backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationQueueService.java

```diff
@@ -102,4 +102,18 @@ public String createAndActivateQueuedOrganization(OrganizationQueueItem queueIte
   public List<OrganizationQueueItem> getUnverifiedQueuedOrganizations() {
     return _orgQueueRepo.findAllNotIdentityVerified();
   }
+
+  public OrganizationQueueItem markPendingOrganizationAsDeleted(
+      String orgExternalId, boolean deleted) {
+    Optional<OrganizationQueueItem> optionalQueueItem =
+        _orgQueueRepo.findUnverifiedByExternalId(orgExternalId);
+
+    if (optionalQueueItem.isEmpty()) {
+      throw new IllegalStateException("This organization doesn't exist");
```

Comment:
nit: this is the message that will show to the Support folks, so we might want to make this a little more explicit than usual user-facing messages: "Organization external id not found in pending org table" or similar.

Reply:
Done!